### ISSUE TITLE
Popped-out reply message still crosses with the email content

### ISSUE THREAD

**Issue by ymolodtsov** [leggett/simplify#335]
I'm using Simplify v2 on a 13' MacBook on its native non-scaled resolution. When I pop out the reply it still crosses with the text. The preferred behavior for me would be to still make both columns more narrow but separate them clearly.

I don't think it'd be too narrow because Superhuman looked OK when doing the same. 

<img width="1552" alt="Screen Shot 2020-09-01 at 18 49 08" src="https://user-images.githubusercontent.com/2446078/91875403-1d526a00-ec84-11ea-9afb-0afb278b0a1f.png">

**Comment by leggett** [comment]
Superhuman as narrower default message width which I think is too narrow. 

I can look into detecting when this happens and doing something about it... but it is tricky.

**Comment by ymolodtsov** [comment]
Yes, it's certainly not ideal in Superhuman and they do a lot of unusual UI juggling to incorporate it but still, seeing the entire text was better.

Of course, even the current behavior is much-much better compared to default Gmail.

**Comment by leggett** [comment]
Note to self on this... one possible improvement here is calculating the left margin for the conversation and action bar so that in cases like this, the conversation is moved over as far to the left as possible. I would need to create --width-mole and change it based on the browser width.

`margin-left: max(0px, calc(100vw - 100px - var(--width-mole) - var(--width-msg))) !important`

**Comment by leggett** [comment]
I started work on this (commented out in _variables.scss). As stated before, it will require mapping the various widths of the mole to the different window heights (how Gmail decides how wide to make the mole) to do properly. 

It might also be annoying if it reflows the message too much so will need to think about when it doesn't make sense to do.
